index.html

```diff
@@ -29,7 +30,61 @@
             </ul>
           </nav>
           <div class="header_btn">
-            <a href="#">Login</a>
+            <button id="modal_open" class="open_btn">Login</button>
+          </div>
+
+          <div class="back-drop" id="back-drop">
+            <form action="main.php">
+              <div class="modal">
+                <div class="modal_close">
+                  <button id="modal_close" class="close-btn">
```

Comment:
```suggestion
                  <button type="button" id="modal_close" class="close-btn">
```
За замовчуванням тип кнопки - submit, відповідно вона викликатиме submit форми, через це і відбувається перенаправлення на сторінку, яка вказана в `action` форми.